node/runtime/pangoro/src/pallets/to_substrate_backing_.rs

```diff
@@ -166,9 +122,10 @@ impl Config for Runtime {
 	type OutboundPayload = ToPangolinMessagePayload;
 	type CallEncoder = PangolinCallEncoder;
 
-	type FeeAccount = RootAccountForPayments;
-	type MessageSender = ToPangolinMessageSender;
+	type MessageNoncer = PangolinMessageNoncer;
```

Comment:
Maybe we can remove `type MessageNoncer` in the `to_substrate_backing` Config and use `type MessagesBridge` works as well.

```rs
fn latest_received_nonce(lane: bp_messages::LaneId) -> bp_messages::MessageNonce {
    BridgePangolinMessages::outbound_latest_received_nonce(lane)
}

fn latest_generated_nonce(lane: bp_messages::LaneId) -> bp_messages::MessageNonce {
    BridgePangolinMessages::outbound_latest_generated_nonce(lane)
}
```

Reply:
O, Ignore this comment. We can't do this after some thought.